android/app/src/main/java/org/electroncash/electroncash3/Main.kt

```diff
@@ -493,6 +505,56 @@ class ChangePasswordDialog : AlertDialogFragment() {
 }
 
 
+class WalletRenameDialog : AlertDialogFragment() {
```

Comment:
This should be called RenameWalletDialog, for consistency with OpenWalletDialog, DeleteWalletDialog, etc. And the layout file should be renamed to match.

Reply:
At the risk of bikeshedding... I had worded it this way round intentionally, given that I'm expecting to add the wallet export functionality too.  I noticed that there were layout files such as "request_detail.xml", "request_list.xml", "transaction_detail.xml", "transaction_list.xml" etc, and their corresponding classes are named similarly.  With this pattern, related files appear next to each other in default alphabetical ordering in directory listings (eg, the transaction related files appear next to each other).  So, I was intending to instead standardise to have "wallet_add", "wallet_open", "wallet_delete", "wallet_export", etc.

This being said, I'll happily adjust to use the existing verb-noun pattern if you still prefer.

Reply:
Either way is fine, as long as it's consistent.

Reply:
Ok.  Probably wasn't worth the fuss, especially since it was a bit more work to standardise the other way than I expected.  But after adjusting, I do think it is neater this way.  I adjusted a handful of other class/layout names into the same format too.  All of the refactoring related to this is in this commit: 4f60bc1c.